casper/src/main/scala/coop/rchain/casper/engine/CasperLaunch.scala

```diff
@@ -60,7 +60,13 @@ object CasperLaunch {
         validatorId <- ValidatorIdentity.fromConfig[F](conf)
         genesis     = approvedBlock.candidate.block
         casper <- MultiParentCasper
-                   .hashSetCasper[F](validatorId, genesis, conf.shardId, conf.finalizationRate)
+                   .hashSetCasper[F](
+                     validatorId,
+                     genesis,
+                     conf.shardId,
+                     conf.finalizationRate,
+                     true
```

Comment:
Please use named parameter, it will be more understandable than just `true`. There are other two places also.

Reply:
Eagle eyes 👍